src/i18n/scripts/intl-imports.js

```diff
@@ -0,0 +1,116 @@
+#!/usr/bin/env node
+const fs = require('fs/promises');
+
+const i18nDir = 'src/i18n';
+const messagesDir = `${i18nDir}/messages`;
+
+async function generateDirectoryIndex(directory) {
+    console.log('generateDirectoryIndex', directory);
+
+    const importLines = [];
+
+    const messagesLines = [];
+
+    try {
+        const files = await fs.readdir(`${messagesDir}/${directory}`, { withFileTypes: true });
+
+        const jsonFiles = files.filter(file => file.isFile() && file.name.endsWith('.json'))
+
+        if (!jsonFiles.length) {
+            console.error(`Skipping ${directory} because no .json translation files were found.`);
+            return;
+        }
+
+        for (const [i, file] of jsonFiles.entries()) {
+            const languageCode = file.name.replace(/\.json$/, '');
+            const languageCodeDash = languageCode.toLowerCase().replace(/_/g, '-')
+            const uniqueMessageVariable = `messages${i}${languageCode}`;
```

Comment:
I'm not sure how I feel about just adding a number to these message var names, it'd be nicer if the name provided more information.

Would it possibly work to use the directory as part of the name instead?